indigo/analysis/toc/base.py

```diff
@@ -377,6 +388,21 @@ def title_without_type(self, item):
 
         return title
 
+    def get_def_element(self, element):
+        """ Uses the refersTo of an element containing a definition to find the matching <def> inside that element.
+            Returns the result of the xpath, which may be an empty list.
+        """
+        term = element.get('refersTo')
```

Comment:
be defensive and assume that this `blockContainer` could appear anywhere in the document, and may not always be related to a term definition. So guard against this being None.